packages/core/src/indexing/local-index-api.ts

```diff
@@ -99,8 +99,8 @@ export class LocalIndexApi implements IndexApi {
       const edges = await Promise.all(
         // For database queries we bypass the stream cache and repository loading queue
         page.edges.map(async (edge) => {
-          let node: StreamState = await this.repository.streamState(edge.node)
-          if (node === undefined) {
+          let node = await this.repository.streamState(edge.node)
+          if (!node) {
             this.logger.warn(`
             Did not find stream state in our state store when serving an indexed query.
```

Comment:
```suggestion
            Did not find stream state for streamid ${edge.node} in our state store when serving an indexed query.
```

Reply:
let's add this too while we're at it so we can tell what's the problematic streamid